### PR TITLE
WIP: Start testing against RHEL 7.3

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

File: test/images/scripts/rhel-7.bootstrap [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 BASE=$(dirname $0)
-$BASE/virt-builder-fedora "$1" rhel-7.2 x86_64 ${SUBSCRIPTION_PATH:-~/.rhel}
+$BASE/virt-install-fedora "$1" x86_64 "http://download-node-02.eng.bos.redhat.com/released/RHEL-7/7.3/Server/x86_64/os/" ${SUBSCRIPTION_PATH:-~/.rhel}
```
Comment (dperpeet) [inline test/images/scripts/rhel-7.bootstrap]: I recommend using the more generic `http://download.eng.bos.redhat.com/released/RHEL-7/7.3/Server/x86_64/os/`, that will automatically redirect to a more specific url.

**Comment by mvollmer** [comment]
There is a 7.3 virt-builder template now.

**Comment by dperpeet** [comment]
> There is a 7.3 virt-builder template now.

Let's use that and disable the beta repos.

**Comment by mvollmer** [comment]
I think this is superseded by #5445, right?

**Pull request closed without merging**